io-engine/src/core/snapshot.rs

```diff
@@ -145,6 +218,22 @@ pub trait SnapshotOps {
 
     /// List Single snapshot details based on snapshot UUID.
     fn list_snapshot_by_snapshot_uuid(&self) -> Vec<VolumeSnapshotDescriptor>;
+
+    async fn create_clone(
+        &self,
+        clone_param: CloneParams,
+    ) -> Result<Lvol, Self::Error>;
```

Comment:
Should Lvol here be Self::Lvol, just like Error?

Reply:
Currently SnapshotOps trait is only for Lvol. Better to keep as Self::Lvol for uniformity. Fixed it.